operator/pkg/ciliumendpointslice/cell.go

```diff
@@ -65,29 +70,34 @@ type Config struct {
 	CESDynamicRateLimitNodes    []string `mapstructure:"ces-dynamic-rate-limit-nodes"`
 	CESDynamicRateLimitQPSLimit []string `mapstructure:"ces-dynamic-rate-limit-qps-limit"`
 	CESDynamicRateLimitQPSBurst []string `mapstructure:"ces-dynamic-rate-limit-qps-burst"`
+	CESDynamicRateLimitConfig   string   `mapstructure:"ces-rate-limits"`
 }
 
 var defaultConfig = Config{
-	CESMaxCEPsInCES:             100,
-	CESSlicingMode:              "cesSliceModeIdentity",
-	CESWriteQPSLimit:            10,
-	CESWriteQPSBurst:            20,
-	CESEnableDynamicRateLimit:   false,
-	CESDynamicRateLimitNodes:    []string{},
-	CESDynamicRateLimitQPSLimit: []string{},
-	CESDynamicRateLimitQPSBurst: []string{},
+	CESMaxCEPsInCES:           100,
+	CESSlicingMode:            "cesSliceModeIdentity",
+	CESDynamicRateLimitConfig: "",
```

Comment:
NIt. It may be worth to display the actual default configuration, rather than an empty string.

Reply:
I removed the default rate limit struct and put the default values here instead.